webapp-django/crashstats/supersearch/tests/test_views.py

```diff
@@ -216,7 +216,9 @@ def mocked_supersearch_get(**params):
         assert '<th scope="col">Bugs</th>' in smart_text(response.content)
         assert "123456" in smart_text(response.content)
         # Test links on terms are existing
-        assert "product=%3DWaterWolf" in smart_text(response.content)
+        assert "build_id=%3D888981" in smart_text(response.content)
```

Comment:
Adding a test for `build_id` here tests the refine link where `use_is_query=True`.